### DISCUSSION TITLE
Module not found: Can't resolve 'exercises' in 'react_frontend/src/exercise-index.js'

### DISCUSSION THREAD

**Discussion by meekhumor** [JdeRobot/RoboticsAcademy#2924] [Q&A] [answered]
## Development Environment  
- Operating System: Windows 10 
- Node.js Version: 20.12.2
- Docker Version: 27.3.1  
- Yarn Version: 1.22.22

I encountered an issue while trying to run the frontend for Robotics Academy. Attempted to run the frontend using `yarn run dev` after reading the [Instructions for Developers](https://github.com/JdeRobot/RoboticsAcademy/blob/humble-devel/docs/InstructionsForDevelopers.md). The following error occurred:
 
![Error](https://github.com/user-attachments/assets/f139cd0e-ae39-4fdc-a382-da269570262e)

Any assistance with this issue would be appreciated.

**Comment by javizqh** [comment, marked as answer]
Can you try with the instructions for Windows that are found [in the user guide](https://jderobot.github.io/RoboticsAcademy/user_guide/). The ones in the developer guide are only for Linux until we solve our remaining issues on Windows.

Also in Windows you must use WSL to run it.

**Comment by meekhumor** [comment]
Thank you for the suggestion. Since I want to contribute, I initially followed the [developer guide](https://github.com/JdeRobot/RoboticsAcademy/blob/humble-devel/docs/InstructionsForDevelopers.md), which provided two options for local setup: 1. Using the automatic script and 2. Using Docker.

I encountered an error when using the automatic script. While Docker allows me to run the project locally but changes, I make in the code are not reflecting. I have WSL2 installed on my Windows machine. 
I'm eager to contribute and would appreciate any guidance or solutions to address these issues on Windows.

**Comment by javizqh** [comment]
I can't help you much as I do not have Windows, but if you follow the installation guide inside WSL2 it should work.
The issues on Windows are already being worked on and hopefully close to being solved.

**Comment by meekhumor** [comment]
Thanks for your help earlier! Just an update: I’ve since switched to Linux, and everything is working fine now.
